roles/openshift_version/meta/main.yml

```diff
@@ -12,6 +12,7 @@ galaxy_info:
   categories:
   - cloud
 dependencies:
+- role: openshift_inventory
```

Comment:
We don't need to bikeshed on this, `openshift_sanitize_inventory` sounds reasonable. Gives out a bit more of the intention without one needing to resort to the README in the role.